atom/browser/api/atom_api_web_contents.cc

```diff
@@ -185,6 +196,11 @@ WebContents::WebContents(const mate::Dictionary& options) {
   AttachAsUserData(web_contents);
   InitWithWebContents(web_contents);
 
+  // Save the preferences.
+  base::DictionaryValue web_preferences;
+  mate::ConvertFromV8(isolate, options.GetHandle(), &web_preferences);
+  new WebContentsPreferences(web_contents, &web_preferences);
```

Comment:
Either style (`new XXX` or `XXX:Create`) seems to be popular in Chromium, I just didn't bother to add a new static method.